### DISCUSSION TITLE
Cannot Connect to Server

### DISCUSSION THREAD

**Discussion by Abateman121** [MDeLuise/plant-it#391] [Q&A]
So i can get to the initial page where it asks for your serverl url and entering it using either the fontend or backend port does not work.  What is supposed to go here.  Based on the fact that your example has ports 8080 i assume it is my backend port i should be using for this.  My port mappings are 12044 for the frontend and 12043 for the backend.  Any guidence on how to move forward?

![image](https://github.com/user-attachments/assets/74a74585-5b7d-4ef7-ab1a-e662d6851af2)

**Comment by cantodorum** [comment]
Did you change the DB password or the Dr root password? They HAVE to be root in order to let the frontend connect to backend

**Reply by dvvincent** [reply]
i had the same problem with the proxmox helper script where I had to guess the backend URL and port (which was 8080). i could then login to the web interface.
